AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -2494,6 +2494,8 @@ full-count.jp##.erase_space-ad
 full-count.jp#?#.l-sidebar > aside.l-sidebar__block:has(> div.l-sidebar-ranking > h2:contains(SAMURAI JAPAN))
 worldstarhiphop.com###marquee-wrapper
 worldstarhiphop.com##.marquee-section
+zeenews.india.com##.field-item > p > a[href="https://zeenews.india.com/live-tv"]
+zeenews.india.com##.field-item > p > iframe[src^="https://zeenews.india.com/live-tv/"]
```

Comment:
Player also may be blocked by URL
`https://zeenews.india.com/live-tv/embed#mute`
Live TV will not be affected
https://zeenews.india.com/live-tv